Update unown configs.

notheowner opened the pull request:
* Remove legacy dynamic route split setting, it's hard coded as true in Dragonite now.
* Add the new scout age limit setting
* Enable logs by default for Dragonite for easier potential troubleshooting.
* Add Golbat tuning section

notheowner committed:
Update configs

HugoDataAnalyst commented:
Isn't the save = true in Dragonite logs irrelevant? Since the AIO already has docker logs?
Everything else indeed.

notheowner commented:
> Isn't the save = true in Dragonite logs irrelevant? Since the AIO already has docker logs? Everything else indeed.

The docker-compose file already binds the volume for it. https://github.com/The-Pokemod-Group/Aegis-All-In-One/blob/main/docker-compose.yaml#L61 Dragonite already log rotates and this provides much easier access to recent and old log files for any troubleshooting. While docker logs on the other side are not that flexible by default and can create disk-exhaustion issues if not configured correctly.

tchavei committed:
Update dragonite_config-default.toml

Agree with everything except the double logging as docker already provides the necessary logs. If people really need them, they can enable them. Leaving them on by default will just take up space.

Same reason I'm updating the README to instruct people to limit docker logging as you suggested on Discord
